### PR TITLE
mkYARA: Initial commit

### LINKED ISSUE #2673: [Tool request] mkYARA
https://github.com/fox-it/mkYARA

### PR DESCRIPTION
**Pull request by sachinkamath**
This PR adds mkYARA and fixes #2673

### CODE REVIEW

File: packages/mkyara/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=mkyara
+_pkgname=mkYARA
+pkgver=3.8147f91
+pkgrel=1
+pkgdesc='Tool to generate YARA rules based on binary code'
```
Comment (Edu4rdSHL) [inline packages/mkyara/PKGBUILD:8]: Add end point to description.

**Comment by Edu4rdSHL** [comment]
Thank you @sachinkamath! Next time do not forge to add the package to lists/to-release.

File: packages/mkyara/PKGBUILD (hunk 1 above)
Comment (sachinkamath) [reply packages/mkyara/PKGBUILD]: @Edu4rdSHL Is there a full list of packaging conventions somewhere that I can read?

**Comment by sachinkamath** [comment]
@Edu4rdSHL Noted! :)

**Pull request merged**